### PR TITLE
tf: drop sleep from revision tests

### PR DESCRIPTION
**Pull request by howardjohn**
**Please provide a description of this PR:**

### CODE REVIEW

File: tests/integration/pilot/revisioncmd/revision_view_test.go [hunk 1]
```diff
@@ -100,10 +99,6 @@ func TestRevisionCommand(t *testing.T) {
 			}
 			builder.BuildOrFail(t)
 
-			// Wait some time for things to settle down. This is very
-			// important for gateway pod related tests.
-			time.Sleep(30 * time.Second)
```
Comment (GregHanson) [inline tests/integration/pilot/revisioncmd/revision_view_test.go:105]: I see this sleep command has been here for the past two years - was there a related change that eliminates the need for this sleep?  Revision code/logic stabilize enough?

Comment (howardjohn) [reply tests/integration/pilot/revisioncmd/revision_view_test.go:105]: I believe it was just a poor use of sleep - I had plannedto refactor to remove it but didn't see anywhere that really needed it. Ill run the tests a few times to verify though

**Comment by howardjohn** [comment]
/test all

failure not related

**Comment by howardjohn** [comment]
ok last one looked real, hold for now

**Comment by howardjohn** [comment]
we should still improve this but I don't have bandwidth

**Pull request closed without merging**